soh/soh/Enhancements/randomizer/randomizer_entrance_tracker.cpp

```diff
@@ -621,18 +619,10 @@ void InitEntranceTrackingData() {
     SortEntranceListByType(destListSortedByType, 1);
 }
 
-void DrawEntranceTracker(bool& open) {
-    if (!open) {
-        if (CVarGetInteger("gEntranceTrackerEnabled", 0)) {
-            CVarClear("gEntranceTrackerEnabled");
-            LUS::RequestCvarSaveOnNextTick();
-        }
-        return;
-    }
-
+void EntranceTrackerWindow::DrawElement() {
     ImGui::SetNextWindowSize(ImVec2(600, 375), ImGuiCond_FirstUseEver);
 
-    if (!ImGui::Begin("Entrance Tracker", &open, ImGuiWindowFlags_NoFocusOnAppearing)) {
+    if (!ImGui::Begin("Entrance Tracker", &mIsVisible, ImGuiWindowFlags_NoFocusOnAppearing)) {
```

Comment:
i feel like this should be handled by `GuiElement` now, we shouldn't ever get in here if `mIsVisible` isn't true (not sure how the `ImGui::End()` stuff works, should probably do some testing

Reply:
based on my testing i think we need these